frontend/awx/access/credentials/CredentialForm.tsx

```diff
@@ -709,8 +815,11 @@ function CredentialTextInput({
                   {t(`Clear`)}
                 </Button>
               ) : null}
+              {buttons}
             </>
-          ) : undefined
+          ) : (
+            buttons
```

Comment:
Can you explain the logic here? What buttons are supposed to render in the instance of `{buttons}` and `(buttons)`?